torch/_dynamo/variables/builder.py

```diff
@@ -539,10 +539,14 @@ class Autotuner:
             result = CustomizedDictVariable.wrap(self, value)
             result.source = self.source
             return self.tx.output.side_effects.track_object_existing(value, result)
+        elif istype(value, set):
+            self.install_guards(GuardBuilder.EQUALS_MATCH)
+            result = ConstantVariable.create(value=value)
```

Comment:
How do we know the set is a constant?  Couldn't it contain non-constant objects?

Reply:
I will restrict this to when its literal